package.json

```diff
@@ -15,7 +15,7 @@
     },
     "dependencies": {
         "ms-rest-azure": "^2.5.9",
-        "ms-rest-js": "^1.0.1",
+        "@azure/ms-rest-js": "~1.8.2",
```

Comment:
Should we restrict the version of `@azure/ms-rest-js` to 1.8.x? Or should we instead use the `^` symbol and allow any version of this package which matches 1.x.x?

Also, should this dependency be added in the global `package.json`? How many packages from this project uses this dependency? Does only `botbuilder-ai` use it? (more info on `^` and `~` symbols [here](https://stackabuse.com/caret-vs-tilde-in-package-json/))